nservicebus/azure/intro-keep-connection-string-safe.include.md

```diff
@@ -0,0 +1,4 @@
+It is possible to accidentally leak sensitive information in the connection string if it's not properly secured. E.g. the information can be leaked if an error occurs when communicating across untrusted boundaries, or if the error information is logged to an unsecured log file.
```

Comment:
@SeanFeldman I only brought ASQ related section into an include file and, as suggested by @yvesgoeleven, linked it from a common place to avoid copy&paste and docu duplication.
I did nothing on this paragraphs, I'll work on it soon. I've read your comments about it somewhere ;)